### PR TITLE
chore: use github action to deploy to npm

### PR DESCRIPTION
**Pull request by tripodsan**
@trieloff trying to deploy the action to runtime with the github action. I don't want to try it out...
i'll test it with the next update to status :-)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -49,32 +49,6 @@ jobs:
...
-
-    - run:
-        name: Semantic Release
-        command: npm run semantic-release
-
-    - run:
-        name: Link Sequences
-        command: npm run deploy-sequences
-
-    - helix-post-deploy/monitoring:
```
Comment (rofe) [inline .circleci/config.yml:73]: Why did you remove the monitoring config?

Comment (tripodsan) [reply .circleci/config.yml:73]: because we can no longer use it with github actions. is this needed, if the sequence number doesn't change?

Comment (tripodsan) [reply .circleci/config.yml:73]: @rofe will there be a problem if we remove this ?

Comment (rofe) [reply .circleci/config.yml:73]: @tripodsan No, I think the current monitoring setup for `@adobe/helix-status` will stay intact as is, but will no longer be updated in the future. Unless we create a dedicated `helix-runtime-status` service as you suggested [here](https://github.com/adobe/helix-status/issues/231#issuecomment-727871096)

**Review by trieloff — APPROVED** [review]

**Comment by rofe** [comment]
But with the monitoring config removed, we won't be able to automatically update the `@adobe/helix-status` monitor to call a newer version of the action anymore (as I mentioned in https://github.com/adobe/helix-status/pull/233#discussion_r525922034). Sorry if that wasn't clear.

**Comment by tripodsan** [comment]
> But with the monitoring config removed, we won't be able to automatically update the `@adobe/helix-status` monitor to call a newer version of the action anymore (as I mentioned in [#233 (comment)](https://github.com/adobe/helix-status/pull/233#discussion_r525922034)). Sorry if that wasn't clear.

if we set the monitor to `...@v8`, then we only need to update the monitor manually after major releases.
